src/pointer-analysis/value_set.cpp

```diff
@@ -1482,7 +1488,7 @@ void value_sett::do_end_function(
   assign(lhs, rhs, ns, false, false);
 }
 
-void value_sett::apply_code(
+void value_sett::apply_code_rec(
```

Comment:
As much as I agree this is a slightly better name: is the renaming essential?

Reply:
The case for this was: it's better to rename this and override the private function than to override the public `apply_code`, thus separating the customisation point (`apply_code_rec`) from the public interface (`apply_code`).